_data/simple-icons.json

```
@@ -15256,6 +15256,11 @@
             "source": "https://styleguide.typo3.com/styleguide/logo",
             "guidelines": "https://styleguide.typo3.com"
         },
+        {
+            "title": "Typst",
+            "hex": "239DAD",
+            "source": "https://typst.app"
```

```suggestion
            "source": "https://typst.app",
            "guidelines": "https://typst.app/legal/brand/"
```
Don't miss the guidelines!